target/linux/ramips/image/mt7621.mk

```diff
@@ -768,6 +768,16 @@ define Device/iptime_a3004ns-dual
 endef
 TARGET_DEVICES += iptime_a3004ns-dual
 
+define Device/iptime_a3004ns-m
+  $(Device/dsa-migration)
+  IMAGE_SIZE := 16128k
+  UIMAGE_NAME := a3004nm
+  DEVICE_VENDOR := ipTIME
+  DEVICE_MODEL := A3004NS-M
+  DEVICE_PACKAGES := kmod-mt7615e kmod-mt7615-firmware kmod-usb3
```

Comment:
`kmod-mt7615e` can be removed now. Reference: https://github.com/openwrt/openwrt/commit/9a07895729a9701a673951a31ceb71f16f0a6fad